doc/api-extensions.md

```diff
@@ -219,12 +219,12 @@ Introduces the ability to rename a volume group by setting "storage.lvm.vg\_name
 Introduces the ability to rename a thinpool name by setting "storage.thinpool\_name".
 
 ## network\_vlan
-This adds a new "vlan" property to "macvlan" network devices.
+This adds a new "vlan" property to "macvlan" and "physical" network devices.
 
 When set, this will instruct LXD to attach to the specified VLAN. LXD
 will look for an existing interface for that VLAN on the host. If one
 can't be found it will create one itself and then use that as the
-macvlan parent.
+macvlan parent if "macvlan" nictype is selected or as it if "physical" nictype is selected.
```

Comment:
I don't really understand why it should be a new extension. Some properties are already shared by a subset of nictypes. For instance, "parent" property can be used with "physical", "bridged" or "macvlan" nictypes. In the present situation, "vlan" has exactly the same meaning for "macvlan" or for "physical" nictypes. I don't see the issue of using the same property as this is actually the same property. Adding a new property would add confusion.